Bookish/Bookish.Web/Models/AddBookViewModel.cs

```diff
@@ -11,15 +11,38 @@ public class AddBookViewModel
         public string Title { get; }
         public string Authors { get; }
         public int Copies { get; }
-        public string ErrorMessage { get; }
+        public IEnumerable<string> ErrorMessage { get; private set; }
 
-        public AddBookViewModel(string isbn, string title, string authors, int copies, string errorMessage)
+        public AddBookViewModel(string isbn, string title, string authors, int copies, bool isbnError, bool copyError)
```

Comment:
You don't need the copyError bool, you can check that here. It's generally preferrable to avoid having too many boolean arguments if possible, as it makes the method invocations harder to read 🙂